developer-experience/recipes/remote-debugging/kubernetes-dashboard/recipes/setup-kubernetes-dashboard.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes Dashboard
```

Comment:
I realised we now have another recipes folder (in addition to the main recipes folder). I wonder if they should simply be under the kubernetes-dashboard folder

Reply:
Agreed, should we keep it under developer-expereince -> remote-debugging -> kubernetes-dashboard?

Reply:
Adjusted